indico/modules/designer/templates/_list.html

```diff
@@ -60,9 +68,43 @@
                 {% endif %}
             {% endif %}
         </td>
+        <td>
+            {% if tpl.registration_form %}
+                {{- pgettext('Designer template linked to registration form', 'Linked to') -}}
+                <a href="{{ url_for('event_registration.manage_regform', tpl.registration_form) }}">
+                    {{ tpl.registration_form.title }}
+                </a>
+                <button class="i-button icon-cross icon-only text-color borderless danger"
+                        title="{% trans %}Unlink registration form{% endtrans %}"
+                        data-href="{{ url_for('.unlink_regform', tpl) }}"
+                        data-method="PATCH"
+                        data-update="#template-list"
+                        data-confirm="{% trans title=tpl.registration_form.title -%}
+                                            Are you sure you want to unlink the registration form '{{ title }}'?
+                                            Any placeholders specific to that registration form will be removed
+                                            from the template.
+                                      {%- endtrans %}">
```

Comment:
Maybe it would look better with a bit less `padding-left`?
![image](https://github.com/indico/indico/assets/27357203/bc6d4820-60b1-48ae-91d6-34fca346de7b)
(here I'm using `0.1em`)